USAGE.md

```diff
@@ -135,7 +135,9 @@ initials: DSH  # strongly suggested
 orcid: 0000-0002-3012-7446  # mandatory
 twitter: dhimmel  # optional
 email: daniel.himmelstein@gmail.com  # suggested
-affiliations: Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania  # strongly suggested
+affiliations:  # as a list, strongly suggested
```

Comment:
We may want to explain why the `affiliations` differs from the other key-value pairs in the paragraph above.  Something about how affiliations are provided one per line so that each author can have one or more of them.